### PR TITLE
Add snapshot of private `api-dispatch-spec` repository

### PR DESCRIPTION
**Pull request by stefan-brus-frequenz**
This commit moves the existing code into a public repository

### CODE REVIEW

**Comment by leandro-lucarella-frequenz** [comment]
This is exactly the tag v0.7.0, right? If so I would mention it in the commit for extra clarity.

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[submodule "submodules/api-common-protos"]
+	path = submodules/api-common-protos
+	url = git@github.com:googleapis/api-common-protos.git
+[submodule "submodules/frequenz-api-common"]
+	path = submodules/frequenz-api-common
+	url = git@github.com:frequenz-floss/frequenz-api-common.git
```
Comment (leandro-lucarella-frequenz) [inline .gitmodules]: Can we use `https` instead? It is less problematic (I had issues having a private key with a password that was locked and was suprised about being asked about my SSH key when pulling from a public repo):

```suggestion
	url = https://github.com/googleapis/api-common-protos.git
[submodule "submodules/frequenz-api-common"]
	path = submodules/frequenz-api-common
	url = https://github.com/frequenz-floss/frequenz-api-common.git
```

**Review by leandro-lucarella-frequenz — COMMENTED** [review]
One only small thing.

**Comment by stefan-brus-frequenz** [comment]
> This is exactly the tag v0.7.0, right? If so I would mention it in the commit for extra clarity.

Updated

**Review by leandro-lucarella-frequenz — APPROVED** [review]

**Pull request merged**